operator/handler/handler.go

```diff
@@ -62,6 +63,10 @@ type PodExtender struct {
 
 // Handle handles requests to create/update Pod and extends it by adding tailing sidecars
 func (e *PodExtender) Handle(ctx context.Context, req admission.Request) admission.Response {
+	if req.Operation == admv1.Delete {
```

Comment:
Please add a more descriptive comment about the situation when this happens and what this `if` is trying to solve